### DISCUSSION TITLE
Admin Login after installation?

### DISCUSSION THREAD

**Discussion by groetzinger** [filipecarneiro/ELabel#23] [Q&A] [answered]
I have installed the Docker-Image and set my admin-password. But how can I login to the admin-panel?
I can't found any username how will match with my password?! 

Thanks for your help!

**Comment by filipecarneiro** [comment, marked as answer]
Hi,

The username is **Admin**, with capital A.

That's defined in the initialize database task. Maybe we should impreve the documentation about this.

Please feel free to contact me if you need any further information.

**Reply by groetzinger** [reply]
Thanks for your answer. I maked a fresh installation again, and now it works!

There is no option to change the password after the installtion, right?
Also there is are only one user for login -> Admin?

**Comment by filipecarneiro** [comment]
Hi @groetzinger,

This solution is using [ASP.NET Core Identity](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity). So, you can change your password, navigating to `/Identity/Account/Manage/ChangePassword`

You can also add more users, registering them on `/Identity/Account/register` but, then you'll have to assign a role in the database... (table `AspNetUserRoles`).

Having an UI to manage users is on the roadmap ;)
